packages/flutter_tools/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   # To update these, use "flutter update-packages --force-upgrade".
   archive: 2.0.11
   args: 1.5.2
-  dwds: 0.7.9
+  dwds: 0.3.3
```

Comment:
Somehow this is downgrading here. The [last change](https://github.com/flutter/flutter/pull/44344) was _upgrades_ the version.